route/build/src/historyFile.f90

```diff
@@ -309,6 +321,15 @@ SUBROUTINE createNC_rch_hru(this, nRch_in, nHRU_in, ierr, message)
         if(ierr/=0)then; message=trim(message)//trim(cmessage); return; endif
       end do
 
+      call put_attr(this%pioFileDesc, ncd_global, 'mizuRoute-version', version ,ierr, cmessage)
```

Comment:
For the CESM coupling case you should probably fill this with model_version from lnd_comp_nuopc.F90

Reply:
Yes, there is a function to get a ROF model version in [rof_comp_nuopc.F90 ](https://github.com/nmizukami/mizuRoute/blob/b76659c8ee7208b8027fbe0b7ef76b5b9c35295c/route/build/cpl/nuopc/rof_comp_nuopc.F90#L401), which is copied in a ROF shared variable named `version` by calling `rofVarSet` [here](https://github.com/nmizukami/mizuRoute/blob/b76659c8ee7208b8027fbe0b7ef76b5b9c35295c/route/build/cpl/nuopc/rof_comp_nuopc.F90#L500). 

I added mizuRoute shared variable `version` in RtmVar.f90, so `model_version` is now copied directly to mizuroute shared variable `version`  in rofVarSet subroutine (RtmVar.f90)